### ISSUE TITLE
SSBL Will Not Compile Due To Multiple Definitions

### ISSUE THREAD

**Issue by davidbitton** [spracing/ssbl#1]
I'm receiving errors on build. It appears that several same-name methods are defined in more than one place. Here is where I found the definitions for -- as an example -- `void HAL_MspInit(void)`:
```
➜  ssbl git:(master) ✗ grep "void HAL_MspInit" * -Hn --color -R
build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal.lst:254: 212:lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal.c **** __weak void HAL_MspInit(void)
build/SPRACINGH7EXTREME/RELEASE/src/main/target/SPRACINGH7EXTREME/stm32h7xx_hal_msp.lst:84:  50:src//main/target/SPRACINGH7EXTREME/stm32h7xx_hal_msp.c **** void HAL_MspInit(void)
lib/main/STM32H7xx_HAL_Driver/Inc/stm32h7xx_hal.h:919:void HAL_MspInit(void);
lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_msp_template.c:57:void HAL_MspInit(void)
lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal.c:212:__weak void HAL_MspInit(void)
src/main/target/SPRACINGH7EXTREME/stm32h7xx_hal_msp.c:50:void HAL_MspInit(void)
➜  ssbl git:(master) ✗
```
Most of what I read points to a malformed autogenerated CubeMX Makefile that causes the multiple includes. I'm on macos 10.15.6.

#### `make` output:
```
➜  ssbl git:(master) make TARGET=SPRACINGH7EXTREME
mkdir -p build/SPRACINGH7EXTREME/RELEASE
AS (default) src/main/startup_stm32h750xx.s
CC (default) src//main/drivers/quadspi.c
CC (default) src//main/drivers/flash_w25q128.c
CC (default) src//main/drivers/light_led.c
CC (default) src//main/target/SPRACINGH7EXTREME/stm32h7xx_hal_msp.c
CC (default) src//main/target/SPRACINGH7EXTREME/target.c
CC (default) src//main/failure.c
CC (default) src//main/system_stm32h7xx.c
CC (default) src//main/main.c
CC (default) src//main/error.c
CC (default) src//main/stm32h7xx_it.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_adc.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_adc_ex.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_cortex.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_dac.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_dac_ex.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_hash.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_i2c.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_i2c_ex.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_qspi.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_rtc.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_rtc_ex.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_ll_sdmmc.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_sd.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_sd_ex.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_spi.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_spi_ex.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_tim.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_tim_ex.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_uart.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_uart_ex.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_wwdg.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_rcc.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_rcc_ex.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_flash.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_flash_ex.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_gpio.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_hsem.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_dma.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_dma_ex.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_mdma.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_pcd.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_pcd_ex.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_rng.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_pwr.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_pwr_ex.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_ll_usb.c
CC (size) lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_ll_delayblock.c
LD src/main/link/stm32_ram_h750_exst.ld build/SPRACINGH7EXTREME/RELEASE/src/main/startup_stm32h750xx.o build/SPRACINGH7EXTREME/RELEASE/src//main/drivers/quadspi.o build/SPRACINGH7EXTREME/RELEASE/src//main/drivers/flash_w25q128.o build/SPRACINGH7EXTREME/RELEASE/src//main/drivers/light_led.o build/SPRACINGH7EXTREME/RELEASE/src//main/target/SPRACINGH7EXTREME/stm32h7xx_hal_msp.o build/SPRACINGH7EXTREME/RELEASE/src//main/target/SPRACINGH7EXTREME/target.o build/SPRACINGH7EXTREME/RELEASE/src//main/failure.o build/SPRACINGH7EXTREME/RELEASE/src//main/system_stm32h7xx.o build/SPRACINGH7EXTREME/RELEASE/src//main/main.o build/SPRACINGH7EXTREME/RELEASE/src//main/error.o build/SPRACINGH7EXTREME/RELEASE/src//main/stm32h7xx_it.o build/SPRACINGH7EXTREME/RELEASE/src/main/target/SPRACINGH7EXTREME/stm32h7xx_hal_msp.o build/SPRACINGH7EXTREME/RELEASE/src/main/target/SPRACINGH7EXTREME/target.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_adc.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_adc_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_cortex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_dac.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_dac_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_hash.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_i2c.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_i2c_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_qspi.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_rtc.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_rtc_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_ll_sdmmc.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_sd.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_sd_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_spi.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_spi_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_tim.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_tim_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_uart.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_uart_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_wwdg.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_rcc.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_rcc_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_flash.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_flash_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_gpio.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_hsem.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_dma.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_dma_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_mdma.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_pcd.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_pcd_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_rng.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_pwr.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_pwr_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_ll_usb.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_ll_delayblock.o
arm-none-eabi-gcc build/SPRACINGH7EXTREME/RELEASE/src/main/startup_stm32h750xx.o  build/SPRACINGH7EXTREME/RELEASE/src//main/drivers/quadspi.o build/SPRACINGH7EXTREME/RELEASE/src//main/drivers/flash_w25q128.o build/SPRACINGH7EXTREME/RELEASE/src//main/drivers/light_led.o build/SPRACINGH7EXTREME/RELEASE/src//main/target/SPRACINGH7EXTREME/stm32h7xx_hal_msp.o build/SPRACINGH7EXTREME/RELEASE/src//main/target/SPRACINGH7EXTREME/target.o build/SPRACINGH7EXTREME/RELEASE/src//main/failure.o build/SPRACINGH7EXTREME/RELEASE/src//main/system_stm32h7xx.o build/SPRACINGH7EXTREME/RELEASE/src//main/main.o build/SPRACINGH7EXTREME/RELEASE/src//main/error.o build/SPRACINGH7EXTREME/RELEASE/src//main/stm32h7xx_it.o build/SPRACINGH7EXTREME/RELEASE/src/main/target/SPRACINGH7EXTREME/stm32h7xx_hal_msp.o build/SPRACINGH7EXTREME/RELEASE/src/main/target/SPRACINGH7EXTREME/target.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_adc.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_adc_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_cortex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_dac.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_dac_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_hash.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_i2c.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_i2c_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_qspi.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_rtc.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_rtc_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_ll_sdmmc.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_sd.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_sd_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_spi.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_spi_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_tim.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_tim_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_uart.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_uart_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_wwdg.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_rcc.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_rcc_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_flash.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_flash_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_gpio.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_hsem.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_dma.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_dma_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_mdma.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_pcd.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_pcd_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_rng.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_pwr.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal_pwr_ex.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_hal.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_ll_usb.o build/SPRACINGH7EXTREME/RELEASE/lib/main/STM32H7xx_HAL_Driver/Src/stm32h7xx_ll_delayblock.o  -g -gdwarf-2 -mcpu=cortex-m7 -mthumb -mfpu=fpv5-d16 -mfloat-abi=hard -specs=nano.specs -specs=nosys.specs -Tsrc/main/link/stm32_ram_h750_exst.ld  -lc -lm -lnosys -lstdc++ -static -Wl,-Lsrc/main/link -Wl,-Map=build/SPRACINGH7EXTREME/RELEASE/SPRACINGH7EXTREME.map,--cref -Wl,--gc-sections -Wl,--print-memory-usage -flto -fuse-linker-plugin -Ofast  -o build/SSBL_SPRACINGH7EXTREME_RELEASE.elf
Memory region         Used Size  Region Size  %age Used
        ITCM_RAM:          0 GB        64 KB      0.00%
        DTCM_RAM:        2064 B       128 KB      1.57%
             RAM:          96 B        64 KB      0.15%
        CODE_RAM:       11012 B     458688 B      2.40%
       EXST_HASH:          64 B         64 B    100.00%
          D2_RAM:          0 GB       256 KB      0.00%
       MEMORY_B1:          0 GB         0 GB       nan%
         QUADSPI:          0 GB         0 GB       nan%
/usr/local/Cellar/arm-none-eabi-gcc/9-2019-q4-major/gcc/bin/../lib/gcc/arm-none-eabi/9.2.1/../../../../arm-none-eabi/bin/ld: build/SPRACINGH7EXTREME/RELEASE/src/main/target/SPRACINGH7EXTREME/stm32h7xx_hal_msp.o: in function `HAL_MspInit':
/Users/davidbbitton/src/ssbl/src//main/target/SPRACINGH7EXTREME/stm32h7xx_hal_msp.c:56: multiple definition of `HAL_MspInit'; build/SPRACINGH7EXTREME/RELEASE/src//main/target/SPRACINGH7EXTREME/stm32h7xx_hal_msp.o:/Users/davidbbitton/src/ssbl/src//main/target/SPRACINGH7EXTREME/stm32h7xx_hal_msp.c:56: first defined here
/usr/local/Cellar/arm-none-eabi-gcc/9-2019-q4-major/gcc/bin/../lib/gcc/arm-none-eabi/9.2.1/../../../../arm-none-eabi/bin/ld: build/SPRACINGH7EXTREME/RELEASE/src/main/target/SPRACINGH7EXTREME/stm32h7xx_hal_msp.o: in function `HAL_QSPI_MspInit':
/Users/davidbbitton/src/ssbl/src//main/target/SPRACINGH7EXTREME/stm32h7xx_hal_msp.c:85: multiple definition of `HAL_QSPI_MspInit'; build/SPRACINGH7EXTREME/RELEASE/src//main/target/SPRACINGH7EXTREME/stm32h7xx_hal_msp.o:/Users/davidbbitton/src/ssbl/src//main/target/SPRACINGH7EXTREME/stm32h7xx_hal_msp.c:85: first defined here
/usr/local/Cellar/arm-none-eabi-gcc/9-2019-q4-major/gcc/bin/../lib/gcc/arm-none-eabi/9.2.1/../../../../arm-none-eabi/bin/ld: build/SPRACINGH7EXTREME/RELEASE/src/main/target/SPRACINGH7EXTREME/stm32h7xx_hal_msp.o: in function `HAL_QSPI_MspDeInit':
/Users/davidbbitton/src/ssbl/src//main/target/SPRACINGH7EXTREME/stm32h7xx_hal_msp.c:162: multiple definition of `HAL_QSPI_MspDeInit'; build/SPRACINGH7EXTREME/RELEASE/src//main/target/SPRACINGH7EXTREME/stm32h7xx_hal_msp.o:/Users/davidbbitton/src/ssbl/src//main/target/SPRACINGH7EXTREME/stm32h7xx_hal_msp.c:162: first defined here
collect2: error: ld returned 1 exit status
make: *** [build/SSBL_SPRACINGH7EXTREME_RELEASE.elf] Error 1
```

**Comment by davidbitton** [comment]
Just an add-on. I renamed `stm32h7xx_hal_msp.c` to `stm32h7xx_hal_msp.c.0`. The code compiled and I installed it to the board. However, despite a formatted SD card installed, still a double-blink.

**Comment by hydra** [comment]
Should be fixed by #3 - reopen if issues found.
